dcm4che-core/src/main/java/org/dcm4che3/error/NoPixelDataException.java

```diff
@@ -0,0 +1,31 @@
+//
+///////////////////////////////////////////////////////////////
+//                C O P Y R I G H T  (c) 2019                //
+//        Agfa HealthCare N.V. and/or its affiliates         //
+//                    All Rights Reserved                    //
+///////////////////////////////////////////////////////////////
+//                                                           //
+//       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF      //
+//        Agfa HealthCare N.V. and/or its affiliates.        //
+//      The copyright notice above does not evidence any     //
+//     actual or intended publication of such source code.   //
+//                                                           //
+///////////////////////////////////////////////////////////////
+//
+package org.dcm4che3.error;
+
+import java.io.IOException;
+
+/**
+ * Exception that indicates that the pixel data was missing and could not be returned.
+ * @author Andrew Cowan (awcowan@gmail.com)
+ */
```

Comment:
I've added an explicit indication of a DICOM file that has no pixel data.  This is to disambiguate the cases where we expect pixels but they do not exist in the file.  In DCM4CHE2 we had to figure it out on our own, and it was error prone to properly identify the issue.

Reply:
Do you want to allow distinguishing between no pixel data at all and no pixel data for a given frame, at least in the message?  It seems like that is an interesting thing to know when things go wrong.

Reply:
Yes.  I have extended the errors that are reported in the latest build:

- NoPixelDataException: Exception that indicates that the pixel data was missing and could not be returned.
- TruncatedPixelDataException: Thrown when there are fewer bytes available in the stream than expected,
- UnavailableFrameException: Thrown when a frame is requested that does not exist in the underlying instance.